phi/playground/playground.py

```diff
@@ -136,6 +136,20 @@ def agent_chat(body: AgentRunRequest):
                 run_response = cast(RunResponse, agent.run(body.message, stream=False))
                 return run_response.model_dump_json()
 
+        @playground_routes.post("/agent/sessions/all")
```

Comment:
@manthanguptaa 
1. Let's add a `body: GetAgentSessionsRequest` that includes an `agent_id` and `user_id`
2. Let's get all sessions from storage using `agent.storage.get_all_sessions(user_id=user_id)`
3. Let's return `session_id` and `session_data` only

So the code will look like:
```
agent: Optional[Agent] = None
for _agent in self.agents:
    if _agent.agent_id == body.agent_id:
        agent = _agent
        break

if agent.storage is None:
    return JSONResponse(status=404, content=f"Agent does not have storage enabled.")

agent_sessions: List[GetAgentSessions] = []
all_agent_sessions: List[AgentSession] = agent.storage.get_all_sessions(user_id=user_id)
for session in all_agent_sessions:
      agent_sessions.append(GetAgentSessions(
          session_id=session.session_id,
          session_data=session.session_data,
      ))
return sessions
```